src/Jackalope/Transport/DoctrineDBAL/Client.php

```diff
@@ -1449,52 +1449,34 @@ public function getNodes($paths)
      */
     private function pathExists($path, $workspaceName = null)
     {
-        return (boolean) $this->getSystemIdForNodePath($path, $workspaceName);
+        return (boolean) $this->getSystemIdForNode($path, $workspaceName);
     }
 
     /**
-     * Get the database primary key for node at path.
+     * Get the database primary key for node.
      *
-     * @param string $path          Path of the node
+     * @param string $identifier    Path of the identifier
      * @param string $workspaceName To overwrite the current workspace
      *
      * @return bool|string The database id
      */
-    private function getSystemIdForNodePath($path, $workspaceName = null)
+    private function getSystemIdForNode($identifier, $workspaceName = null)
```

Comment:
I have merged this method with ``getSystemIdForNodeUuid()`` so that it automatically determines if its UUID or not ..